### PR TITLE
Update for Bob

### LINKED ISSUE #178: Bob_test.sh tests need fixing
- The shouting with umlauts test is broken.
- Also, they aren't umlauts.
- Also, other tracks don't seem to have these two tests.
### Requesting permission to remove these two tests in a PR

### PR DESCRIPTION
**Pull request by guygastineau**
#### After finally getting my local repo okay again
1. Brought the bob_test.sh into compliance with the `canonical-data` - sort of
   - there is one extra test in the bash bob test. It test for no argument being treated like silence. I think it is fine to leave it in the test.
   - There is one test commented out because it behaves differently on mac an linux.
2. Created the .meta/version file with contents `1.2.0`
3. Added a nested `if` in the **original** `example.sh` to get it to pass the forceful question test.
   - I also added a comment above the nested `if` to imitate the style found in `example.sh`. 

closes #178 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: exercises/bob/bob_test.sh [hunk 1]
```diff
@@ -212,7 +196,7 @@
 # This is commented out because it behaves differently on linux and mac
```
Comment (budmc29) [inline exercises/bob/bob_test.sh]: I think it's better to remove this test altogether, but it's your call.

File: exercises/bob/bob_test.sh [hunk 2]
```diff
@@ -1,23 +1,23 @@
 #!/usr/bin/env bats
 
 @test "stating something." {
-  #skip
+  ##skip
```
[the remark this answers is not in the document]
Comment (guygastineau) [reply exercises/bob/bob_test.sh]: yep I forgot to to `:%s/#skip/skip/g` in vim.  thanks @budmc29

File: exercises/bob/bob_test.sh [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
   run bash bob.sh "Tom-ay-to, tom-aaaah-to."
 
   [ "$status" -eq 0 ]
   [ "$output" = "Whatever." ]
 }
 
 @test "shouting" {
-  skip
+  #skip
```
[the remark this answers is not in the document]
Comment (guygastineau) [reply exercises/bob/bob_test.sh]: Do you mean only the first test should be commented?

**Comment by guygastineau** [comment]
1. fixed the commenting on `skip` in the test
2. removed the commented out test

File: exercises/bob/bob_test.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (budmc29) [reply exercises/bob/bob_test.sh]: Yes.

**Comment by budmc29** [comment]
Thank you for the PR.

**Pull request merged**